### ISSUE TITLE
Is this SDK permanently closed

### ISSUE THREAD

**Issue by shanxiaofan** [Emotiv/community-sdk#379]
Is this SDK permanently closed？

**Comment by bchaperon** [comment]
Hello @shanxiaofan 
As mentioned in the README file, this SDK is longer under development.
Please use the new Cortex API instead.
https://emotiv.gitbook.io/cortex-api/
https://github.com/Emotiv/cortex-v2-example

**Comment by shanxiaofan** [comment]
May I ask if this SDK can still be used

**Comment by bchaperon** [comment]
If you really want to use it then of course you can :)
But it is not recommended because this SDK is not maintained or supported anymore.
Bugs won't be fixed, and new Emotiv hardware won't be supported.

Is there any reason why you prefer to use this SDK instead of switching to the new Cortex API?

**Comment by shanxiaofan** [comment]
Yes, I hope to continue using the SDK. And I have a question, why isn't there a Build_example for Windows. Or, to put it another way, where the Windows edk is on your computer.

**Comment by shanxiaofan** [comment]
./community-sdk-master/examples_extra_prime contains only Build_example of the Mac system
